src/som/vmobjects/SInvokable.java

```diff
@@ -65,6 +68,9 @@ public SInvokable(final SSymbol signature,
     this.invokable = invokable;
     this.callTarget = invokable.createCallTarget();
     this.embeddedBlocks = embeddedBlocks;
+    if (VmSettings.TRACK_SNAPSHOT_ENTITIES) {
```

Comment:
Is this necessary?

We have the `StructuralProbe` mechanism in the parser.
Could that be used instead?

Structural probes have `recordNewMethod(SInvokable method)`.

Reply:
SInvokable information could be used from the probes if i can change it to a Map. For the SClasses it looks different, I don't see a connection from tracked mixins to the SClasses.

Reply:
?

There is no connection between mixins and classes. Mixins are class bodies. Classes are a mixin + a super class + mixins + an identity.
A class group is a mixin + a super class + mixins but without identity.